Sources/SWBUtil/FSProxy.swift

```diff
@@ -397,70 +380,11 @@ class LocalFS: FSProxy, @unchecked Sendable {
     /// - parameter recursive: If `false`, then the parent directory at `path` must already exist in order to create the directory.  If it doesn't, then it will return without creating the directory (it will not throw an exception).  If `true`, then the directory hierarchy of `path` will be created if possible.
     func createDirectory(_ path: Path, recursive: Bool) throws {
         // Try to create the directory.
-        #if os(Windows)
         do {
             return try fileManager.createDirectory(atPath: path.str, withIntermediateDirectories: recursive)
         } catch {
             throw StubError.error("Could not create directory at path '\(path.str)': \(error)")
         }
-        #else
-        let result = mkdir(path.str, S_IRWXU | S_IRWXG | S_IRWXO)
-
-        // If it succeeded, we are done.
-        if result == 0 {
-            return
-        }
-
-        // If the failure was because something exists at this path, then we examine it to see whether it means we're okay.
-        if errno == EEXIST {
-            var destinationExists = false
-            if isDirectory(path) {
-                // If the item at the path is a directory, then we're good.  This includes if it's a symlink which points to a directory.
-                return
-            }
-            else if isSymlink(path, &destinationExists) {
-                // If the item at the path is a symlink, then we check whether it's a broken symlink or points to something that is not a directory.
-                if destinationExists {
-                    // The destination does exist, so it's not a directory.
-                    throw StubError.error("File is a symbolic link which references a path which is not a directory: \(path.str)")
```

Comment:
Test expectations need to be updated to accommodate these specific error strings going away.

Reply:
Ah, thanks!

Reply:
Hmm, I don't see a good way to handle the check. There is no guarantee that the message is going to be identical across platforms and Foundation implementations :(

Reply:
I think it's fine if we just check that it _does_ fail in the expected cases without checking the specific error string.